content/docs/developing-operators/getting-started.md

```diff
@@ -93,16 +91,16 @@ At the same time, `deploy` plan is the most important plan within your operator
 
 Plans allow operators to see what the operator is currently doing, and to visualize the broader operation such as for a config rollout. The fixed structure of that information meanwhile makes it straightforward to build UIs and tooling on top.
 
-## Params File
+# Params File
 
 `params.yaml` defines all parameters that can be used to customize your operator installation.
 They are described in more detail in the [document about operator parameters](operator-parameters.html).
 
-## Templates
+# Templates
 
 Everything that is placed into the templates folder is treated as a template and passed on to the KUDO controller for rendering. KUDO uses the [Sprig template library](https://godoc.org/github.com/Masterminds/sprig) for server side templates rendering during installation. Thanks to Sprig you can use tens of different functions inside your templates. Some of them are inherited from [go templates](https://godoc.org/text/template), some of them are defined by [Sprig](https://godoc.org/github.com/Masterminds/sprig) itself. See their documentation for reference.
 
-### Variables Provided by KUDO
+## Variables Provided by KUDO
 
 - `.OperatorName` - name of the operator the template belongs to
```

Comment:
More details in `templates.md`, we should at least add an `etc.` after the variables, if we want to show them here at all.